interface/therapy_groups/therapy_groups_views/listGroups.php

```diff
@@ -126,12 +126,24 @@
             <tbody>
             <?php foreach ($therapyGroups as $group) : ?>
                 <tr>
+                    <?php
+                    if($_SESSION['language_direction'] == 'rtl'){
+                        $date_start_date = date('d/m/Y',strtotime(text($group['group_start_date'])));
+                        $date_end_date = date('d/m/Y',strtotime(text($group['group_end_date'])));
```

Comment:
Appears that you want to use following function (note that library is included in every script):
https://github.com/openemr/openemr/blob/master/library/formatting.inc.php#L20